administration/troubleshooting.md

```diff
@@ -0,0 +1,311 @@
+[0] trace: [1658953439.096256447, {"type"=>1, "trace_id"=>"trace.5", "plugin_instance"=>"dummy.0", "plugin_alias"=>"dummy_0", "records"=>[{"timestamp"=>1658953439, "record"=>{"dummy"=>"dummy_0", "key_name"=>"foo", "key_cnt"=>"1"}}], "start_time"=>1658953439, "end_time"=>1658953439}]
+[0] trace: [1658953439.096277177, {"type"=>2, "start_time"=>1658953439, "end_time"=>1658953439, "trace_id"=>"trace.5", "plugin_instance"=>"record_modifier.0", "records"=>[{"timestamp"=>1658953439, "record"=>{"dummy"=>"dummy_0", "key_name"=>"foo", "key_cnt"=>"1", "powered_by"=>"calyptia"}}]}]
+[0] trace: [1658953439.096291774, {"type"=>2, "start_time"=>1658953439, "end_time"=>1658953439, "trace_id"=>"trace.5", "plugin_instance"=>"nest.2", "records"=>[{"timestamp"=>1658953439, "record"=>{"dummy"=>"dummy_0", "powered_by"=>"calyptia", "data"=>{"key_name"=>"foo", "key_cnt"=>"1"}}}]}]
+[0] trace: [1658953439.096298678, {"type"=>3, "trace_id"=>"trace.5", "plugin_instance"=>"dummy.0", "plugin_alias"=>"dummy_0", "records"=>[{"timestamp"=>1658953439, "record"=>{"dummy"=>"dummy_0", "powered_by"=>"calyptia", "data"=>{"key_name"=>"foo", "key_cnt"=>"1"}}}], "start_time"=>1658953439, "end_time"=>1658953439}]
+^C[2022/07/27 20:24:00] [engine] caught signal (SIGINT)
+[2022/07/27 20:24:00] [ warn] [engine] service will shutdown in max 5 seconds
+[0] trace: [1658953440.096253908, {"type"=>1, "trace_id"=>"trace.6", "plugin_instance"=>"dummy.0", "plugin_alias"=>"dummy_0", "records"=>[{"timestamp"=>1658953440, "record"=>{"dummy"=>"dummy_0", "key_name"=>"foo", "key_cnt"=>"1"}}], "start_time"=>1658953440, "end_time"=>1658953440}]
+[2022/07/27 20:24:01] [ info] [engine] service has stopped (0 pending tasks)
+[0] trace: [1658953440.096277973, {"type"=>2, "start_time"=>1658953440, "end_time"=>1658953440, "trace_id"=>"trace.6", "plugin_instance"=>"record_modifier.0", "records"=>[{"timestamp"=>1658953440, "record"=>{"dummy"=>"dummy_0", "key_name"=>"foo", "key_cnt"=>"1", "powered_by"=>"calyptia"}}]}]
+[0] trace: [1658953440.096291459, {"type"=>2, "start_time"=>1658953440, "end_time"=>1658953440, "trace_id"=>"trace.6", "plugin_instance"=>"nest.2", "records"=>[{"timestamp"=>1658953440, "record"=>{"dummy"=>"dummy_0", "powered_by"=>"calyptia", "data"=>{"key_name"=>"foo", "key_cnt"=>"1"}}}]}]
+[0] trace: [1658953440.096298913, {"type"=>3, "trace_id"=>"trace.6", "plugin_instance"=>"dummy.0", "plugin_alias"=>"dummy_0", "records"=>[{"timestamp"=>1658953440, "record"=>{"dummy"=>"dummy_0", "powered_by"=>"calyptia", "data"=>{"key_name"=>"foo", "key_cnt"=>"1"}}}], "start_time"=>1658953440, "end_time"=>1658953440}]
+```
+### Analysis of a single trace record
+
+Here I analyze a single trace record from a filter event to explain the meaning of each field in detail. I chose a filter record since it includes the most details of all the record types.
```

Comment:
As always, split on to separate lines.

```suggestion
Here we analyze a single record from a filter event to explain the meaning of each field in detail. 
We chose a filter record since it includes the most details of all the record types.
```